cdhweb/people/models.py

```diff
@@ -166,11 +166,12 @@ def not_students(self):
         '''Filter out graduate and undergraduates based on PU status'''
         return self.exclude(pu_status__in=self.student_pu_status)
 
-    def faculty_affiliates(self):
-        '''Faculty affiliates based on PU status and Project Director
-        project role.'''
-        return self.filter(pu_status='fac',
-                           user__membership__role__title='Project Director')
+    def affiliates(self):
+        '''Faculty and staff affiliates based on PU status and Project Director
+        project role. Excludes CDH staff.'''
+        return self.filter(pu_status__in=('fac', 'stf'),
```

Comment:
Might be nice to have a variable for the affiliate statuses, but I guess it may not get us much if we're not using it anywhere else.